### PR TITLE
draft of pages for Subcommittees

### PR DESCRIPTION
**Pull request by katieannemills**
There has been some discussion recently of providing a landing page for the Mentorship Subcommittee on the SWC website (see https://github.com/swcarpentry/board/pull/102, ping @raynamharris); this PR adds:
- the aforementioned Mentorship page, with text pulled verbatim from the PR linked above
- a template for general Subcommittee pages

~~Still todo: how to link to these? I would do a 'Subcommittees & Task Forces' link in the 'Get Involved' dropdown that navigates to a landing page for the whole subcom / tf program, which then links to the specific groups.~~ **Edit:** <-- implemented below

### CODE REVIEW

**Comment by katieannemills** [comment]
oh and actually cc @ChristinaLK - discuss at next mentorship meeting / or async comments from the committee?

**Comment by raynamharris** [comment]
Yeah! Thanks for getting this started @BillMills

**Comment by ChristinaLK** [comment]
@BillMills, async comments sounds fine to me.  :)  Email the mentoring list?  Important person to loop in is probably @jduckles or whoever is in charge of the website...

**Comment by jduckles** [comment]
Looks good to me. Flagging this as in-progress until we're ready to merge.

**Comment by katieannemills** [comment]
Latest push adds:
- a top level subcom & tf page: essentially an edit of @raynamharris's blog post, plus a table of all existing subcoms and task forces
- a link in the 'Get Involved' dropdown to said top-level page.

The only thing left is to make sure there's at least a stub for all currently active subcommittees in the top-level table; atm only Mentoring is listed, but I think we have several other active groups that should be on here - let me know who's missing, then we should be good to merge.

**Comment by jduckles** [comment]
@BillMills, @raynamharris, @ChristinaLK I think this fell through the cracks, is it ready to merge?

**Comment by katieannemills** [comment]
Other subcoms / tfs to list? Currently only mentoring is present. Also since this has hung for a while, someone should rebase this on their local machine and make sure it still works properly.

**Comment by raynamharris** [comment]
@BillMills  I feel like assessment should be up there, but that committee/role is sorta in flux. Also, maybe the in-prep guides proposal...

**Comment by ChristinaLK** [comment]
I would add the maintainers (as the lesson committee) and possibly the AMY developers?  They're not a sub-committee officially, but function a bit like one (or a task force)

**Comment by ErinBecker** [comment]
@BillMills This looks really nice. Thanks for getting this started. I second what @raynamharris  and @ChristinaLK said earlier about adding the assessment and maintainers subcommittees and the AMY developers and guide projects as task forces. 

I don't know how active these groups still are, but the board repo does list a few other subcommittees (finance, partner relations, lesson development and communication). If these are groups that are open to new members, I think they should also be listed - to make it clear that there are many ways to be involved in our community. 

Also, if I understand the structure correctly, most/all of these subcommittees and task forces are open also to people who are involved in the DC side of things. If that's correct, maybe we could modify the language a bit just to mention DC once or twice and I can cross post from the DC website?

**Comment by ErinBecker** [comment]
Some slightly more concrete suggestions:

1) Can we move the "Current Subcommittees and Task Forces" section above the one about proposing new groups? I think people more likely to join something that already exists than go through the process of proposing a new group. Want to get their attention on that at top.

2) I like the "activities" and "goals" summaries. Some suggested text for other groups:
- Assessment / Activities:  Work with Deputy Director of Assessment to assess effectiveness of activities, including educational efficacy, community engagement, and other metrics.  
- Maintainers / Activities: ???
- Finance / Activities: Recommend financial policies and budgets to support the mission and goals of the Software Carpentry Foundation. Review financial activities and create monthly reports.  
- Partner Relations / Activities: Manage stakeholder engagement, attract new partnerships and pursue funding.  
- Lesson Organization and Development / Activities: Directs development, organization and maintenance of curriculum materials.
- Communications / Activities: Facilitate communication between Steering Committee and various stakeholders.
- AMY developers / Goals: ???  
- Guides / Goals: Help instructor trainees progress through the checkout process. Provide personalized mentorship and support.  

Most of this text is stolen verbatim from [here](https://github.com/swcarpentry/board/blob/master/subcommittees/subcommittees.md). May need to be more succinct.

**Comment by ChristinaLK** [comment]
@BillMills if this is more than you intended to bite off, I'm happy to fill in skeleton pages if necessary.  Or I'll wrangle someone else to help.  :)

**Comment by katieannemills** [comment]
**TLDR**: committees and task forces require:
- a lead organizer
- a Steering Committee liaison
- a set of goals / roadmap

before being ratified as a standing project - many of the groups listed above have none of these.

**Detail version**:
Thanks @ErinBecker @ChristinaLK @raynamharris - and no need to pull anyone else in, I can implement some of these suggestions in the next few days, but I am -1 to including all the groups listed above; the Steering Committee adopted this new structure because many of those groups are completely defunct, and this is our reboot of a more intentional, community driven and action-oriented organizing principle for these efforts.

Including Maintainers (as a committee) and AMY (as a task force) sounds good in principle, but there is procedure that needs to be addressed first, as listed in the existing text; at a minimum, both groups need to identify a Steering Committee liaison who they will report to the SC through, and a set of goals and/or services they wish to provide. @gvwilson, you have the most regular contact with both these groups; do you have thoughts on how to proceed here? I volunteer to liaise with both groups if they want to be organized in this capacity.

Groups like assessment and finance haven't to date had any (or at least not consistently many) people participating, and we want to avoid placeholders; another requirement for these groups is that they have active and engaged leadership. Re: comms, that's up to @weaverbel to organize as our defacto comms czar; in general, what we most need from each group listed on the website is a chair / leader actively putting effort in that we can direct volunteers to.

**Comment by katieannemills** [comment]
Latest commit makes language agnostic between SWC / DC and moves 'current' ahead of 'proposed', per @ErinBecker's suggestions.

Recommend getting [African Guides Task Force](https://github.com/swcarpentry/board/issues/118) integrated (to be ratified this Thursday at SC), and something at least interim for Maintainers and AMY before merge.

**Comment by raynamharris** [comment]
I second @BillMills's -1 above. We should go forward with the mentoring committee pages. Other pages can be added ad hoc when they are ready to face the public (something we determined as a group during the mentoring committee meeting).

**Comment by ErinBecker** [comment]
Agreed with @BillMills  and @raynamharris to add the African Guides Task Force, Maintainers and AMY before merge. There was discussion at the Mentoring meeting yesterday about reviving the Assessment group, but this will likely be a task for the new assessment person who comes on board with DC next month. 

It's exciting to see these changes. Making the work done by these groups more visible and accessible to the public is great! (And if it helps with recruitment, all the better!)

**Comment by ErinBecker** [comment]
Is this ready to be rolled out as is? Or if not, anything I can do to help get it up and running? I'd really like to see this info be more accessible. I had a heck of a time even finding a current list of SWC steering committee members the other day.

**Comment by katieannemills** [comment]
Very shortly. Since adding on maintainers and amy, I needed to make sure a fair and even-handed process was getting applied and the right communication responsibilities were getting set up. @gvwilson has this covered for maintainers, @maneesha can help with AMY, and @raynamharris and @ChristinaLK are pulling something together for Mentorship. I need to merge a pr from Greg this evening, then I will validate and merge this one in. Thanks for everyone's feedback!

**Comment by katieannemills** [comment]
Last two commits from @gvwilson and I have this almost ready for prime-time; minimal last blocker is that the task forces need contact points for people who want to get involved. Anything the task force leaders like is fine by me - could be a mailing list, could be a repo to open an issue in, could be a twitter account to tweet at - anything you find convenient that gives people a clear way to make first contact. So specifically:
- @anelda, we need a point of contact for the African Guides group.
- @maneesha and @gvwilson, same for AMY.

Once that contact information is in place, I will be r+ to merge on this. I'd like to see actual pages for the task forces, too, but we can do that in a separate pull, in the interests of getting this merged soon.

**Comment by wking** [comment]
On Tue, Jun 21, 2016 at 09:21:56PM -0700, Bill Mills wrote:

> … minimal last blocker is that the task forces need contact points
> for people who want to get involved…

It seems asymmetric to have task-force “Get Involved” inlined in the
subcom.html task-force table but to have that delegated to the
per-subcomittee pages (and not in the subcom.html subcommittee table).
The “anything goes” approach is fine with me, in which case we should
probably always delegate to the per-group page (where there is more
space to talk about the option(s) they've chosen).  And it lets you
punt to this follow-up PR:

> … I'd like to see actual pages for the task forces, too, but we can
> do that in a separate pull, in the interests of getting this merged
> soon.

**Comment by weaverbel** [comment]
@BillMills @anelda @ErinBecker  I am happy to be point of contact for African Guides. Sorry for delay in putting my hand up for this - still playing catch up after my little hiatus ...

People can ping me on Twitter (cloudaus), github (weaverbel) or you can list an email for me in the format `name` AT `domainname`

**Comment by katieannemills** [comment]
@wking I nominally agree, but folks want this merged faster than we are likely to have all the text pulled together for pages for each task force - consider it an interim solution. **Edit**: tell you what - since physicists can't resist a good symmetry argument, we can symmetrize things by going the other way, and adding a contact column on both tables.

Thanks, @weaverbel! Twitter sounds easiest, let's go with that.

**Comment by anelda** [comment]
Thanks @weaverbel @BillMills! This looks good.

**Comment by katieannemills** [comment]
Alright, in lieu of other suggestions I'm pointing at AMY's issue tracker for their get involved link; building the site on top of gh-pages HEAD looks good to me, so I am r+ to merge this now if there are no objections.

**Comment by katieannemills** [comment]
ping - merge plz!
